usecase/parser/msg_exec_test.go

```diff
@@ -78,6 +97,22 @@ var _ = Describe("ParseMsgCommands", func() {
 					-1,
 				),
 			))
+
+			innerCmd := cmds[1]
+			Expect(innerCmd.Name()).To(Equal("CreateMsgSend"))
+
+			untypedInnerEvent, _ := innerCmd.Exec()
+			createMsgSendEvent := untypedInnerEvent.(*event.MsgSend)
+
+			Expect(json.MustMarshalToString(createMsgSendEvent)).To(Equal(
+				strings.Replace(
+					regex.ReplaceAllString(expectedInnerMsg, ""),
+					"{UUID}",
+					createMsgSendEvent.UUID(),
+					-1,
+				),
+			))
+
```

Comment:
Can we have 1-2 more examples of MsgExec with a more complicated inner message? Want to make sure the parser is correct for diff message types.

Reply:
Sure! Let me check if I could find more MsgExec related tx.

Reply:
@calvinaco Hi, I just added one more MsgExec example, whose inner msg is `MsgDelegate`.